client/src/modules/stock/StockForm.service.js

```diff
@@ -125,5 +126,57 @@ function StockFormService(StockItem, Store, AppCache, Session, $timeout) {
     return Object.keys(this.cache).length > 0;
   };
 
+  /**
+   * @method catchDuplicatedLots
+   *
+   * @description
+   * this method catch duplicated row and emit notification on the row
+   */
+  StockForm.prototype.catchDuplicatedLots = function catchDuplicatedLots(_store) {
+    const ROW_ERROR_FLAG = bhConstants.grid;
```

Comment:
I think this should be:
```js
const { ROW_ERROR_FLAG } = bhConstants.grid;
```

Reply:
Oops... it seems like the highlight works only with this `const ROW_ERROR_FLAG = bhConstants.grid;` syntax.

Reply:
Did you also make this change in the controller?

See: https://github.com/IMA-WorldHealth/bhima/pull/3926/files#r333052727